### ISSUE TITLE
isl-0.20 download failing in Docker build

### ISSUE THREAD

**Issue by xandramax** [VCVRack/rack-plugin-toolchain#12]
Attempting to run `docker build --tag rack-plugin-toolchain:1 .` from PowerShell on Windows 10 is failing here:

```
#16 89.71 Now configured for "x86_64-w64-mingw32"
#16 89.72 make[1]: Leaving directory '/home/build/rack-plugin-toolchain'
#16 89.72 CT_PREFIX="/home/build/rack-plugin-toolchain/local" ct-ng build
#16 89.72 make[1]: Entering directory '/home/build/rack-plugin-toolchain'
[INFO ]  Performing some trivial sanity checks
[INFO ]  Build started 20211014.191122
[INFO ]  Building environment variables
[WARN ]  Directory '/home/build/src' does not exist.
[WARN ]  Will not save downloaded tarballs to local storage.
[EXTRA]  Preparing working directories
[EXTRA]  Installing user-supplied crosstool-NG configuration
[EXTRA]  =================================================================
[EXTRA]  Dumping internal crosstool-NG configuration
[EXTRA]    Building a toolchain for:
[EXTRA]      build  = x86_64-pc-linux-gnu
[EXTRA]      host   = x86_64-pc-linux-gnu
[EXTRA]      target = x86_64-w64-mingw32
[EXTRA]  Dumping internal crosstool-NG configuration: done in 0.12s (at 00:01)
[INFO ]  =================================================================
[INFO ]  Retrieving needed toolchain components' tarballs
[EXTRA]    Retrieving 'zlib-1.2.11'
[EXTRA]    Verifying SHA512 checksum for 'zlib-1.2.11.tar.xz'
[EXTRA]    Retrieving 'gmp-6.1.2'
[EXTRA]    Verifying SHA512 checksum for 'gmp-6.1.2.tar.xz'
[EXTRA]    Retrieving 'mpfr-4.0.2'
[EXTRA]    Verifying SHA512 checksum for 'mpfr-4.0.2.tar.xz'
[EXTRA]    Retrieving 'isl-0.20'
[ERROR]    isl: download failed
[ERROR]
[ERROR]  >>
[ERROR]  >>  Build failed in step 'Retrieving needed toolchain components' tarballs'
[ERROR]  >>        called in step '(top-level)'
[ERROR]  >>
[ERROR]  >>  Error happened in: CT_Abort[scripts/functions@487]
[ERROR]  >>        called from: CT_DoFetch[scripts/functions@2103]
[ERROR]  >>        called from: CT_PackageRun[scripts/functions@2063]
[ERROR]  >>        called from: CT_Fetch[scripts/functions@2174]
[ERROR]  >>        called from: do_isl_get[scripts/build/companion_libs/121-isl.sh@16]
[ERROR]  >>        called from: do_companion_libs_get[scripts/build/companion_libs.sh@15]
[ERROR]  >>        called from: main[scripts/crosstool-NG.sh@648]
[ERROR]  >>
[ERROR]  >>  For more info on this error, look at the file: 'build.log'
[ERROR]  >>  There is a list of known issues, some with workarounds, in:
[ERROR]  >>      https://crosstool-ng.github.io/docs/known-issues/
[ERROR]  >>
[ERROR]  >> NOTE: Your configuration includes features marked EXPERIMENTAL.
[ERROR]  >> Before submitting a bug report, try to reproduce it without enabling
[ERROR]  >> any experimental features. Otherwise, you'll need to debug it
[ERROR]  >> and present an explanation why it is a bug in crosstool-NG - or
[ERROR]  >> preferably, a fix.
[ERROR]  >>
[ERROR]  >>  If you feel this is a bug in crosstool-NG, report it at:
[ERROR]  >>      https://github.com/crosstool-ng/crosstool-ng/issues/
[ERROR]  >>
[ERROR]  >>  Make sure your report includes all the information pertinent to this issue.
[ERROR]  >>  Read the bug reporting guidelines here:
[ERROR]  >>      http://crosstool-ng.github.io/support/
[ERROR]
[ERROR]  (elapsed: 2:02.23)
```

**Comment by cschol** [comment]
Yes. This is a known issue in crosstool-NG. This will definitely be fixed in the `v2` branch by working around the issue. I will check into how easy it is to backport the fix to the `v1` branch.

**Comment by cschol** [comment]
Fixed in cf15f75104c3eb6ef745078e1b631a3e41367da2.
